ferrocene/doc/user-manual/src/rustfmt/cli.rst

```diff
@@ -8,23 +8,37 @@
 
    .. cli:option:: --check
 
-      Run in 'check' mode. Exits with 0 if input is formatted correctly. Exits with 1 and prints a diff if formatting is required.
+      Run in 'check' mode.
 
-   .. cli:option:: --emit [files|stdout|coverage|checkstyle|json]
+      Exits with status code 0 if input is formatted correctly.
+      Exits with status code 1 and prints a diff if formatting is not correct.
 
```

Comment:
Let's add examples to all options, like we do for `rustc`.

```suggestion
      .. code-block::

         $ # well-formatted file
         $ rustfmt --check src/main.rs
         $ echo $?
         0
         $ # not well-formatted file
         $ rustfmt --check src/main.rs 
         Diff in /home/urhengulas/Documents/krate/src/main.rs at line 1:
          fn main() {
         -    println!(
         -        "Hello, world!"
         -    );
         +    println!("Hello, world!");
          }
         $ echo $?
         1
```

Reply:
This example doesn't seem to provide much information. We already say above what the exit codes say. The only useful thing might be the diff, but even then it's described above.

Reply:
I thought that we should include and example for each, since we do that for the `rustc` docs[^1]. And those examples also mostly show what was described in the text.

[^1]: https://public-docs.ferrocene.dev/main/user-manual/rustc/cli.html